### ISSUE TITLE
Move Table 1 to supplement (possibly with MS11 discussion entirely?)

### ISSUE THREAD

**Issue by wkumler** [wkumler/MesoscopeMetabolomicsManuscript#6]
Reviewer 1 | 04 Sep 2024 | 01:18:
Table 1, the format is not appropriate

Reviewer 2 | 04 Oct 2024 | 12:20:
Chapter 4.1.1 It is strange to read an entire chapter on an outlier. It is really one measurement that could be easily contaminated (e.g. one single zooplankton organism on the filter would change everything) and I would not recommend to write an entire chapter about the metabolome, risking to discuss artifacts and not data important for the large picture. Is table 1 required.

**Comment by wkumler** [comment]
Originally I included this because I hoped that someone later would realize what had gone wrong - as reviewer 2 suggests it could be a zooplankton or something on the filter. I thought it could be neat to then have a list of the indicators of said zooplankton like I was hoping it would be with Tricho. If it's a total tangent though then it should be removed and the narrative cleaned up as we did when submitting to mSystems.
